learning-bitmark/working-with-bitmarks/payment-for-bitmark-cli.md

```diff
@@ -0,0 +1,261 @@
+    ```
+
+<br>
+
+* Install the Bitmark Wallet
+
+    ```shell
+        $cd bitmark-wallet/command/bitmark-wallet
+
+        $go install
+    ```
+
+<br>
+
+* Create Bitmark Wallet's config file
+
+    ```shell
+        $ touch <CONFIG DIR>/bitmark-wallet/bitmark-wallet.conf
+    ```
+
+    *Example:*
+    ```shell
+        $ mkdir ${HOME}/.config/bitmark-wallet
+        $ touch ${HOME}/.config/bitmark-wallet/bitmark-wallet.conf
+    ```
+
+<br>
+
+* Copy the content from the [sample config file](samples/bitmark-wallet.conf) to the created config file above
+
+<br>
+<br>
+
+### Connect the Bitmark Wallet with bitcoind and litecoind
+
+* Download the [rpcauth.py](samples/rpcauth.py)
+
+<br>
+
+* Get the ltc and btc rpcauth, rpc username, and rpc password by running
+
+    ```shell
+        $ python3 rpcauth.py localhost
+    ```
+
+<br>
+
+* Copy the btc and ltc username and password and paste to the corresponding field in the bitmark-wallet.conf
+
+    *Example:*
+    ```
+        agent {
+          ltc {
+            type = "daemon"
+            node = "localhost:19332"
+            user = "sfkrvwbjdqtmwrewxfjkowrs"
+            pass = "8D0PFG7t_lhfdhPa1IWs-AZxuHI41X3q9S8QRKScIrw="
+          }
+          btc{
+            type = "daemon"
+            node = "localhost:18332"
+            user = "mhvvmfdcucaustjpzuiavokq"
+            pass = "Ft6lkX3wB_9I0kioqGkkxuVxD072bMUkCnwTlmjgKoc="
+          }
+        }
+    ```
+
+<br>
+
+* Copy the corresponding rpcauth and past to the  litecoin and bitcoin config files
+
+    *Example:*
+    ```
+    #In litecoin.conf:
+
+        # authentication
+        rpcauth=sfkrvwbjdqtmwrewxfjkowrs:13b9cd202435e8696f050fa899ad64$a24cb045e3cd0fac9d5659a7073aea9787bac818989c08d5acb1465c4db79b11
+    ```
+
+    ```
+    # In bitcoin.conf:
+
+        # authentication
+        rpcauth=mhvvmfdcucaustjpzuiavokq:3088a1682d42acc444abe51e8d3c3e$ec512a3538ada3b4ef55ea8dd1eadc82d274c908eda094c04924e50220e808ee
+    ```
+
+<br>
+
+> **NOTE:** Transactions on the Bitmark testnet blockchain need to be paid by the testnet coins. To do so, enable the testnet by adding `testnet=1` field to the bitcoind/litecoind config files.
+
+<br>
+<br>
+
+### Run the Bitmark Wallet
+
+* Run the coin servers
+
+    ```shell
+        # bitcoind
+        $ bitcoind -datadir=<bitcoind config dir>
+    ```
+    ```shell
+        # litecoind
+        $ litecoind -datadir=<litecoind config dir>
+    ```
+
+    *Example:*
+    ```shell
+        # bitcoind
+        $ bitcoind -datadir=${HOME}/.config/bitcoin
+    ```
+    ```shell
+        # litecoind
+        $ litecoind -datadir=${HOME}/.config/litecoin
+    ```
+
+<br>
+
+* Init the bitmark wallet
+
+    ```shell
+        $  bitmark-wallet --conf=<conf file> init
+    ```
+
+    *Example:*
+    ```shell
+        # bitcoind
+        $ bitmark-wallet --conf=${HOME}/.config/bitmark-wallet/test/bitmark-wallet.conf init
+    ```
+
+<br>
+
+* Sync the wallet with the coins
+
```

Comment:
Since this isn't a standard BTC/LTC command it'd be nice to have a little explanation of what "syncing" is.

Reply:
Added the explanation.